<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quit_window</class>
 <widget class="QMainWindow" name="quit_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/quitbackground.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_no">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_no {	
	background-color: rgb(255,255,255);
	color: rgb(0,0,0);
	border-radius:5px;
}
QPushButton#btn_no:hover{	
	color: rgb(255,0,0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(255, 255, 255), stop:1  rgb(255, 255, 255));
}
QPushButton#btn_no:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>HAYIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_yes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_yes {	
	background-color: rgb(255,255,255);
	color: rgb(0,0,0);
	border-radius:5px;
}
QPushButton#btn_yes:hover{	
	color: rgb(255,0,0);
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(255, 255, 255), stop:1  rgb(255, 255, 255));
}
QPushButton#btn_yes:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>EVET</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_question">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>501</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Çıkmak istediğinize &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;emin misiniz ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
